### PR TITLE
issue-913

### LINKED ISSUE #913: Create README File for app\Css
The README should details briefly what the folder contains, and any notable characteristics of the folder.

### PR DESCRIPTION
**Pull request by PkHutch**
Fixes #913.

### CODE REVIEW

File: app/Css/INFORMATION.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Information
+===========
+
+Summary
+-------
+
+This folder contains the Haskell for defining any page's CSS. The convention
+is that the file name is the same as the page for which it defines the CSS.
```
Comment (david-yz-liu) [inline app/Css/INFORMATION.md]: Please give an example (e.g., `graph`)

File: app/Css/About.hs [hunk 2]
```diff
@@ -8,8 +14,7 @@ import Prelude hiding ((**))
 import Data.Monoid
 import Css.Constants
 
-{- aboutStyles
- - Generates CSS for the about page. -}
+-- |Returns the CSS for the about page.
```
Comment (david-yz-liu) [inline app/Css/About.hs]: Don't say `returns` to describe nullary (0-parameter) functions. Just say that this value *is* the CSS for the about page.

Comment (PkHutch) [inline app/Css/About.hs]: See commit cd95a85.

**Comment by PkHutch** [comment]
@david-yz-liu , headers added, Haddock auto-generates a table of contents which is nice. The reason I switched the order of constants is so they more closely reflected the order they appeared in the rest of the file.

**Comment by david-yz-liu** [comment]
Thanks, @PkHutch!

**Pull request merged**